server/helper/CatchErrors.js

```diff
@@ -0,0 +1,84 @@
+
+/**
+ * @class CatchErrors
+ *
+ * @description catches 500 errors across StackOverFlow-Lite.
+ */
+class CatchErrors {
+  /**
+        * @description -This method takes care of any error resulting from connecting to a database
+        *
+        * @param {object} err - The error object catched by this handler
+        * @param {object} res - The response object sent back to the client
+        *
+        * @returns {object} - returns the response object
+        *
+        * @memberOf CatchErrors class
+        * @static
+        */
+  static catchDatabaseConnectionError(err, res) {
```

Comment:
Instead of using `err` as a variable name, use `error`, so it's clear to anyone reading your code exactly what it stands for. Do the same for `res` and `req`.

Reply:
I will change all at once